### PR TITLE
0.9.0

### PR DESCRIPTION
**Pull request by MoOx**
0.9 is almost ready, we still miss

- [x] #276 - @thangngoc89
- [x] #262 - @MoOx
- [x] #269 - @MoOx
- [x] #279  - @thangngoc89
- [x] #325 - @thangngoc89
- [x] #324 - @thangngoc89
- [x] check docs
- [x] tldr section in CHANGELOG
- [x] allow install from github #203

### CODE REVIEW

**Comment by thangngoc89** [comment]
Huray. It's the 300th PR/issue

**Comment by thangngoc89** [comment]
And coverage only decrease 3%

**Comment by MoOx** [comment]
We also need to backport master changes (docs) to this branch to be able to merge. Not sure what is the best. Both branches are protected. Should we unprotect next and rebase next on master?

**Comment by thangngoc89** [comment]
Yup. Maybe. Can we cherrypick master commit to next ? And then force push next to master. Some branches is still tracking next branch. Force push next is a bad idea.

**Comment by MoOx** [comment]
I think forcepushing master is a bad idea. next is "unstable" so rebasing this one (changing commit hashes) does not look like a problem.

**Comment by thangngoc89** [comment]
Neither nyc exclude pattern is working. I'll try different pattern locally. 

https://travis-ci.org/MoOx/statinamic/builds/116061867#L817
https://travis-ci.org/MoOx/statinamic/builds/116063248#L793

**Comment by MoOx** [comment]
Wtf, it was working locally.

**Comment by thangngoc89** [comment]
it dosen't work for me

**Comment by thangngoc89** [comment]
Found it. Preparing a PR

**Comment by thangngoc89** [comment]
Oh yes. Coverage is now > 70 %

**Comment by MoOx** [comment]
We are working hard for it. I am on #262 so we can end and merge this PR :)
Btw maybe I will try to handle #203 before cutting this release (idea: allow #203, try this branch on a project, migrate a project, add migration commit in the changelog, make the release ^^).

**Comment by thangngoc89** [comment]
I had a WIP branch for #203 https://github.com/MoOx/statinamic/commit/93d2ba078aee41b363676c8e0b16852543a45090

It's not working yet.

**Comment by thangngoc89** [comment]
FYI, can't install from github on next branch.

```
$ node -v && npm -v
v5.8.0
3.8.3
```

**Comment by MoOx** [comment]
Hum, just got the "target of null" error, it was working a moment ago...

**Comment by thangngoc89** [comment]
I got no such file or directory 

Full log: https://gist.github.com/thangngoc89/d10f3ae5773be5da31c1

**Comment by MoOx** [comment]
@thangngoc89 your error looks like a npm/git error. Try clean the node_modules folder.

**Comment by thangngoc89** [comment]
@MoOx I ran the test on a clean folder each time

**Comment by thangngoc89** [comment]
Install from github took longer than it supposed to be. I suggest we ship 0.9.0 right now and review it later.

**Comment by MoOx** [comment]
I pushed some modifications, the "npm install" is failling, so I only kept for now the "transpile" step, with a warning + doc on the setup page and subscribed to the issue on npm.

**Comment by thangngoc89** [comment]
OK. That's fine now. We can switch to the original method when npm's bug resolved. Can we ship this now ?

**Comment by MoOx** [comment]
Finishing changelog note + example and will cut a release.

File: src/content-loader/validator.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const fieldTypes = {
   feedsOptions: "object",
   feeds: "object",
   description: "object",
+  defaultHead: "object",
```
Comment (thangngoc89) [inline src/content-loader/validator.js]: Ouch. I don't even know we have this option. 

Haha. Good catch

Comment (MoOx) [inline src/content-loader/validator.js]: Will add tests for this later :)

**Comment by MoOx** [comment]
@thangngoc89 any idea why codecov isn't working?

**Comment by MoOx** [comment]
nvm, it seems to work atm.

**Comment by thangngoc89** [comment]
@MoOx no rock star merge badge ?

**Comment by MoOx** [comment]
haha. I will remove this stupid label. I will simplify those soon based.

**Pull request merged**